backend/Dockerfile

```diff
@@ -71,8 +72,11 @@ RUN echo -e 'gunicorn backend.wsgi:application --bind 0.0.0.0:8000\n' > entrypoi
 
 FROM base AS demo
 
+ADD nginx.conf /etc/nginx/nginx.conf
```

Comment:
Is nginx really needed here? I think that Nginx should be spawned as a separate docker container, just like the docker-compose.yml was doing.

Reply:
For convenience, they are now sharing an image.